tests/integration/server/courseInstance/courseInstance.controller.test.ts

```diff
@@ -1303,4 +1307,118 @@ describe('CourseInstance API', function () {
       });
     });
   });
+
+  describe('GET /room-schedule', function () {
```

Comment:
As far as I can see, we're never actually testing the `CourseInstanceService#getRoomSchedule` method itself, just the controller behavior. This isn't the worst thing, since the method is mainly relying on TypeORM to assemble the final data object, but it might be worth adding some test for the ordering behavior to ensure that the final output will render appropriately in the client. (I gave myself a headache looking back at the tests I wrote for [the other schedule method][1], so I don't think we need to go into _that_ level of detail).

[1]: https://github.com/seas-computing/course-planner/blob/6d237d1cf2ed1b7e9dc5677ad9723080e1866cf6/tests/integration/server/courseInstance/courseInstance.controller.test.ts#L230-L597

Reply:
Thanks for your guidance on this Jonathan. I added tests to check the ordering of instructors in commit [45f7693](https://github.com/seas-computing/course-planner/pull/562/commits/45f76939c9fa15f62687ec3e435705836302d520).